tests/prometheus/basic.py

```diff
@@ -202,3 +203,31 @@ def assert_publish_metrics(self, metrics_manager: Any):
                 status="success",
             ),
         ]
+
+    async def test_one_registry_for_some_middlewares(
```

Comment:
Can we make this test in-memory?

Reply:
Also, seems like we shouldn't publish a message and/or even create a broker to validate target behavior `assert middleware_1._metrics_container is middleware_2._metrics_container`

Reply:
done, simplified the test

I didn’t see the point in in memory testing in a simplified test